src/sources/forus-platform/scss/_common/landing.scss

```diff
@@ -2167,7 +2166,7 @@ ui-view {
     @media screen and (max-width: 1700px) {
         &.block-navbar {
             .navbar-logo {
-                padding: 20px 20px 20px 0;
+                padding: 20px 25px 20px 0;
```

Comment:
In combination with my earlier feedback, adding this much padding: 30px 30px should do the job.